files/etc_cron.daily_datagov-docker

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# Clean up unnecessary docker files/images/builds
+
+docker image prune
```

Comment:
Should be safe to prune all the things not currently attached to running containers.

```suggestion
docker container prune
docker volume prune
docker image prune
```